examples/arduino-uno/src/bin/uno-ext-interrupt.rs

```diff
@@ -63,6 +63,6 @@ fn main() -> ! {
 
     loop {
         blink_for_range(0..10, &mut leds);
-        blink_for_range(10..0, &mut leds);
+        blink_for_range((0..10).rev(), &mut leds);
```

Comment:
This is a change of behavior, see this [playground](https://play.rust-lang.org/?version=stable&mode=debug&edition=2021&gist=270f7fdddc13442c407a6b46dadc8380).

Arguably the original code doesn't make much sense here anyway but I think it is better to fix it properly instead.  I see two options:

1. Just dropping the "count down" line and leaving only the "count up".  This should be functionally equivalent to the previous version.
2. Or simplifying this `blink_for_range()` function by not passing the range.  I think this would make the whole example a lot more readable...

Reply:
@mutantbob, can you take care of this when you get the chance?  Otherwise the changes look good and I'd like to get them merged :)  If you can't get to it right now, I can offer to fix it up and pull in your changes manually.  If that's okay with you.

Reply:
It looks like the original writer intended the code to work as in this patch, and yes it is cluttering up the example.